JavaScriptSPA/authRedirect.js

```diff
@@ -0,0 +1,57 @@
+
+"use strict";
+
+// Create the main myMSALObj instance
+// configuration parameters are located at authConfig.js
+const myMSALObj = new Msal.UserAgentApplication(msalConfig);
+
+// Register Callbacks for Redirect flow
+myMSALObj.handleRedirectCallback(authRedirectCallBack);
+
+function authRedirectCallBack(error, response) {
+    if (error) {
+        console.log(error);
+    } else {
+        if (response.tokenType === "id_token" && myMSALObj.getAccount() && !myMSALObj.isCallback(window.location.hash)) {
```

Comment:
Lets sync offline on this, we do this check in the library too and looks redundant for me. But I do understand the value in adding it here. Helps to spend some time to establish a `redirect` API usage pattern for `msal js`. I can back port this to dev samples too if needed.

Reply:
you're probably right. I removed this and so far there are no issues -took this bit uncritically from dev samples.